tests/wps_restapi/test_jobs.py

```diff
@@ -39,6 +46,8 @@
 if TYPE_CHECKING:
     from typing import Iterable, List, Tuple, Union
 
+TEST_DATE_INTERVALL = generate_test_datetimes()
```

Comment:
You can move this inside the class setup as `cls.datetime_intervals` and refer to it in tests with `self.datetime_interval`.